### ISSUE TITLE
Android : Crash on "Delete offline stories..."

### ISSUE THREAD

**Issue by shuvashish76** [samuelclay/NewsBlur#1705]
**Describe the bug:**
App crashing when click on "Delete offline stories..."

**Crash logs:**

```
FATAL EXCEPTION: main
Process: com.newsblur, PID: 24149
java.lang.NullPointerException: Attempt to invoke virtual method 'void com.newsblur.database.BlurDatabaseHelper.deleteStories()' on a null object reference
 at com.newsblur.fragment.SettingsFragment.deleteOfflineStories(SettingsFragment.java:45)
 at com.newsblur.fragment.SettingsFragment.lambda$onCreatePreferences$0(SettingsFragment.java:33)
 at com.newsblur.fragment.SettingsFragment.$r8$lambda$3GV46yVQADJGbiEeSXKu2skiicI(Unknown Source:0)
 at com.newsblur.fragment.SettingsFragment$$ExternalSyntheticLambda0.onPreferenceClick(Unknown Source:2)
 at androidx.preference.Preference.performClick(Preference.java:1184)
 at androidx.preference.Preference.performClick(Preference.java:1166)
 at androidx.preference.Preference$1.onClick(Preference.java:181)
 at android.view.View.performClick(View.java:6294)
 at android.view.View$PerformClick.run(View.java:24774)
 at android.os.Handler.handleCallback(Handler.java:790)
 at android.os.Handler.dispatchMessage(Handler.java:99)
 at android.os.Looper.loop(Looper.java:164)
 at android.app.ActivityThread.main(ActivityThread.java:6518)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```
App version: 11.3.1 (203)

**Comment by sictiru** [comment]
Thanks for reporting! The issue was fixed earlier today and it will be available in the next release.

**Comment by sictiru** [comment]
Fixed in NewsBlur v12.0
